src/bundle/config/wysiwyg.ts

```diff
@@ -271,6 +271,19 @@ export const wMermaidItemData: WToolbarSingleItemData = {
     isEnable: (e) => e.actions.createMermaid.isEnable(),
 };
 
+export const wQuoteLinkItemData: WToolbarSingleItemData = {
```

Comment:
This approach to creating items is outdated, use `toolbarPresets` instead.
https://github.com/gravity-ui/markdown-editor/blob/main/src/modules/toolbars/items.tsx